peachjam/urls.py

```diff
@@ -90,6 +92,8 @@
         AboutPageView.as_view(),
         name="about",
     ),
+    path("folders/", CreateFolderView.as_view(), name="folders"),
+    path("save-document/", SaveDocumentView.as_view(), name="save_document"),
```

Comment:
saved-documents
saved-documents/<int:pk>
saved-documents/<int:pk>/delete

Reply:
we split up the views to be more inline with this pattern, code feels much simpler